specification/registry/xr.xml

```diff
@@ -5748,6 +5748,35 @@ maintained in the default branch of the Khronos OpenXR GitHub project.
+
+        <interaction_profile name="/interaction_profiles/oppo/mr_controller_oppo" title="OPPO MR Controller">
+            <user_path path="/user/hand/left"/>
+            <user_path path="/user/hand/right"/>
+            <component user_path="/user/hand/left"    subpath="/input/x/click"              type="XR_ACTION_TYPE_BOOLEAN_INPUT"/>
+            <component user_path="/user/hand/left"    subpath="/input/x/touch"              type="XR_ACTION_TYPE_BOOLEAN_INPUT"/>
+            <component user_path="/user/hand/left"    subpath="/input/y/click"              type="XR_ACTION_TYPE_BOOLEAN_INPUT"/>
+            <component user_path="/user/hand/left"    subpath="/input/menu/click"           type="XR_ACTION_TYPE_BOOLEAN_INPUT"/>
+            <component user_path="/user/hand/left"    subpath="/input/y/touch"              type="XR_ACTION_TYPE_BOOLEAN_INPUT"/>
+            <component user_path="/user/hand/left"    subpath="/input/heartrate_oppo/value" type="XR_ACTION_TYPE_FLOAT_INPUT"/>
+            <component user_path="/user/hand/right"   subpath="/input/a/click"              type="XR_ACTION_TYPE_BOOLEAN_INPUT"/>
+            <component user_path="/user/hand/right"   subpath="/input/a/touch"              type="XR_ACTION_TYPE_BOOLEAN_INPUT"/>
+            <component user_path="/user/hand/right"   subpath="/input/b/click"              type="XR_ACTION_TYPE_BOOLEAN_INPUT"/>
+            <component user_path="/user/hand/right"   subpath="/input/b/touch"              type="XR_ACTION_TYPE_BOOLEAN_INPUT"/>
+            <component user_path="/user/hand/right"   subpath="/input/home/click"           type="XR_ACTION_TYPE_BOOLEAN_INPUT" system="true"/>
+            <component                                subpath="/input/squeeze/value"        type="XR_ACTION_TYPE_FLOAT_INPUT"/>
+            <component                                subpath="/input/trigger/touch"        type="XR_ACTION_TYPE_BOOLEAN_INPUT"/>
+            <component                                subpath="/input/trigger/value"        type="XR_ACTION_TYPE_FLOAT_INPUT"/>
+            <component                                subpath="/input/thumbstick/x"         type="XR_ACTION_TYPE_FLOAT_INPUT"/>
+            <component                                subpath="/input/thumbstick/y"         type="XR_ACTION_TYPE_FLOAT_INPUT"/>
+            <component                                subpath="/input/thumbstick/click"     type="XR_ACTION_TYPE_BOOLEAN_INPUT"/>
+            <component                                subpath="/input/thumbstick/touch"     type="XR_ACTION_TYPE_BOOLEAN_INPUT"/>
+            <component                                subpath="/input/grip/pose"            type="XR_ACTION_TYPE_POSE_INPUT"/>
+            <component                                subpath="/input/aim/pose"             type="XR_ACTION_TYPE_POSE_INPUT"/>
+            <component                                subpath="/input/battery/value"        type="XR_ACTION_TYPE_FLOAT_INPUT"/>
```

Comment:
I think this is a custom component so it also needs `_oppo` as a suffix: no other vendor has battery level in their actions.

Reply:
@rpavlik Thanks for the suggestion. Per you comment in another reply, I removed the battery path to be compatible with other interaction profile. We will use battery value on system side (or system shell) only. Thanks